.github/workflows/cyclonus-netpol-test.yaml

```diff
@@ -0,0 +1,101 @@
+name: Cyclonus network policy test
+
+on:
+  schedule:
+    # run once a day at midnight
+    - cron: '0 0 * * *'
+
+env:
+  KIND_VERSION: v0.9.0
+  KIND_CONFIG: .github/kind-config.yaml
+  CONFORMANCE_TEMPLATE: examples/kubernetes/connectivity-check/connectivity-check.yaml
+  TIMEOUT: 2m
+  LOG_TIME: 30m
+
+jobs:
+  preflight-clusterrole:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check pre-flight clusterrole
+        run: |
+          cd install/kubernetes/cilium/templates
+          echo "Checking for differences between preflight and agent clusterrole"
+          diff \
+             -I '^[ ]\{2\}name: cilium.*' \
+             -I '^Keep file in synced with.*' \
+             -I '{{- if.*' \
+             cilium-agent-clusterrole.yaml \
+             cilium-preflight-clusterrole.yaml
+
+  cyclonus-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Precheck generated connectivity manifest files
+        run: |
+          make -C examples/kubernetes/connectivity-check fmt
+          make -C examples/kubernetes/connectivity-check all
+          git diff --exit-code || (echo "please run 'make -C examples/kubernetes/connectivity-check all' and submit your changes"; exit 1)
+
+      - name: Build docker images
+        run: |
+          make docker-cilium-image
+          make docker-operator-generic-image
```

Comment:
The images are no longer built this way, please check how this file is doing:
https://github.com/cilium/cilium/blob/master/.github/workflows/kind-1.19.yaml#L21-L28

https://github.com/cilium/cilium/blob/master/.github/workflows/kind-1.19.yaml#L42-L47

Reply:
👍 okay I just removed that step and copy/pasted the other two steps in its place, 🤞 that will work !